ports/esp8266/modules/_boot.py

```diff
@@ -3,11 +3,11 @@
 import uos
 from flashbdev import bdev
 
-try:
-    if bdev:
+if bdev:
+    try:
         uos.mount(bdev, '/')
-except OSError:
-    import inisetup
-    inisetup.setup()
+    except AttributeError:
```

Comment:
This should at least catch both OSError and AttributeError.  Probably best to just catch anything via a simple `except: ...`.

Reply:
I don't know in which case a `OSError` happens, but i change this to only `except:`